app/services/coms.js

```diff
@@ -448,7 +448,7 @@ export default Service.extend({
             this.updateUsername(message);
             break;
           case 'presence':
-            Logger.debug('Presence update:', message.actor['@id'], message.object.presence, message.object.status);
+            this.getSockethubPlatformFor(message.context).handlePresenceUpdate(message);
```

Comment:
Why not just use `this.get(message.context).handlePresenceUpdate()`? Also, isn't this XMPP-specific?